projects/admin/src/app/record/detail-view/person-detail-view/person-detail-view.component.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * RERO ILS UI
+ * Copyright (C) 2019 RERO
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, version 3 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import { Component, OnInit } from '@angular/core';
+import { DetailRecord } from '@rero/ng-core/lib/record/detail/view/detail-record';
+import { Observable } from 'rxjs';
+
+@Component({
+  selector: 'admin-person-detail-view',
+  templateUrl: './person-detail-view.component.html',
+  styles: []
+})
+export class PersonDetailViewComponent implements DetailRecord, OnInit {
+
+    /** Observable resolving record data */
+    record$: Observable<any>;
+
+    /** Resource type */
+    type: string;
+
+    /** Record */
+    record: object;
+
+  constructor() { }
+
+  ngOnInit() {
+    this.record$.subscribe(record => {
```

Comment:
seems overkill: keep record$ and use record$.metadata in the template, the alternative is to use a get method.

Reply:
done